src/main/webapp/WEB-INF/views/admin/mechanics/add.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title th:text="#{view.index.title}">Dodaj mechanika</title>
+    <meta th:replace="fragments/header :: head"/>
+    <script src="/resources/js/mechanics.js" th:src="@{/resources/js/mechanics.js}"></script>
+    <script src="/resources/js/form-validation.js" th:src="@{/resources/js/form-validation.js}"></script>
+</head>
+<body>
+<div th:replace="fragments/header :: header">&nbsp;</div>
+<div class="container">
+    <div class="jumbotron text-center jumbo-form">
+        <h3>Dodaj mechanika:</h3>
+        <form id="addMechanicForm" name="addPersonForm" class="form-horizontal" action="#" th:object="${mechanic}" onsubmit="return addMechanic();">
```

Comment:
Ten formularz wygląda analogicznie jak formularz do dodawania klienta, może da radę zrobić to jako fragment w Thymeleafie i nie powtarzać tego samego w kilku miejscach?